packages/bvaughn-architecture-demo/components/console/hooks/useTerminalHistory.ts

```diff
@@ -1,16 +1,19 @@
 import { RecordingId } from "@replayio/protocol";
 import { useCallback } from "react";
 
-import useLocalStorage from "bvaughn-architecture-demo/src/hooks/useLocalStorage";
+import { CONSOLE_SETTINGS_DATABASE } from "bvaughn-architecture-demo/src/contexts/ConsoleFiltersContext";
+import useIndexedDB from "bvaughn-architecture-demo/src/hooks/useIndexedDB";
 
 export default function useTerminalHistory(
   recordingId: RecordingId,
   maxHistoryLength: number = 20
 ): [expressionHistory: string[], addExpression: (value: string) => void] {
-  const [expressionHistory, setExpressionHistory] = useLocalStorage<string[]>(
-    `${recordingId}::expressionHistory`,
-    []
-  );
+  const { value: expressionHistory, setValue: setExpressionHistory } = useIndexedDB<string[]>({
+    database: CONSOLE_SETTINGS_DATABASE,
+    initialValue: [],
+    recordName: recordingId,
```

Comment:
Why is this named "recording name"? Seems like we use "recording id" everywhere (even GraphQL). Seems simpler / less confusing to call it what it is.

Reply:
It's `recordName`, as in 'the record in the database".  You wrote that variable name in your original WIP commit :)

Reply:
Oh wow I completely misread "record" as "recording" 😆  Sorry!